### ISSUE TITLE
× TypeError: path.split is not a function

### ISSUE THREAD

**Issue by Jihane-Raoui** [toofaniCoder/material-ui-rhf#1]
helloooo
there is a conflict with the version 7 of react hook form 
thanks for replying

**Comment by toofaniCoder** [comment]
Its aslo happened to me some days ago
Are you set control={control}

**Comment by Jihane-Raoui** [comment]
Yes I did. I tried also to do it using Controller (because a split pasing error is displaying when I used "register" ) but also the same problem.

**Comment by toofaniCoder** [comment]
can you post here some screenshots of your code and erros also
